app/views/homepage/_location_bar.haml

```diff
@@ -8,12 +8,17 @@
     function initialize() {
       var input = document.getElementById('q');
       var autocomplete = new google.maps.places.Autocomplete(input);
+      autocomplete.setTypes(['geocode']);
 
       google.maps.event.addListener(autocomplete, 'place_changed', function(event){
         var place = autocomplete.getPlace();
-        if(place != null && place.name != "") {
-          console.log('name:', place.name, '\naddress: ', place.formatted_address, '\ncoordinates: ', place.geometry.location.toUrlValue());
-          document.getElementById('lc').value = place.geometry.location.toUrlValue();
+        if(place != null) {
+          if( place.geometry != null ) {
+            console.log('name:', place.name, '\naddress: ', place.formatted_address, '\ncoordinates: ', place.geometry.location.toUrlValue());
```

Comment:
These have been here for testing purposes - to check what kind of data we can get and pass to Rails. 
Both console.logs should be removed, but I think the correct place is to remove those when we decide what data we want to pass to the controller. (Currently this passes just location LatLng, and we want to pass at least bounding box too (NorthEast & SouthWest corners)